source/_posts/Improving-your-Azure-Search-performance.md

```diff
@@ -5,24 +5,24 @@ tags:
 - Azure
```

Comment:
alright ben er doorheen, echt goeie blog dibran!

ik zou mijn pr niet accepteren, maar misschien is er wat van de feedback waar je wat aan hebt.
Inhoudelijk erg goed opgesteld met een fijne flow.

de tips bedenk ik me nu zouden misschien iets meer terug kunnen grijpen op de blog, maar als personen daar zijn dan heb je ze al overtuigd /genoeg geleerd Lachebekje
​​​​​​​